config.py

```diff
@@ -7,14 +7,15 @@ class Config:
     SSL_DISABLE = False
     SQLALCHEMY_COMMIT_ON_TEARDOWN = True
     SQLALCHEMY_RECORD_QUERIES = True
+    DEBUG=False
```

Comment:
I dont think so. just i copied from an example on the net. I thought that
if you put it on False, you would see nothing on the screen, but dont.

2015-02-07 5:25 GMT-03:00 Gastón Avila <notifications@github.com>:

> In config.py
> <https://github.com/avilaton/gtfseditor/pull/69#discussion_r24288061>:
>
> > @@ -7,14 +7,15 @@ class Config:
> >      SSL_DISABLE = False
> >      SQLALCHEMY_COMMIT_ON_TEARDOWN = True
> >      SQLALCHEMY_RECORD_QUERIES = True
> > +    DEBUG=False
>
> Is DEBUG=False here this necessary? Why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/avilaton/gtfseditor/pull/69/files#r24288061>.
>